src/guides/v2.4/graphql/interfaces/customizable-option-interface.md

```diff
@@ -236,6 +236,15 @@ The following query returns information about the customizable options configure
           sort_order
           option_id
         }
+        ... on CustomizableFieldOption {
+            textField: value {
```

Comment:
I believe that the "textField" definition here might not be passed by the Webonyx validator (the example #2 is still correct). Also, please not that the response does not correspond the request after extending of this example.

Thanks